sample-token/Makefile

```diff
@@ -0,0 +1,15 @@
+#################
+#   NEARswap    #
+
+
+include ../Makefile_common.mk
+
+export NCLP_ACC=beta-1.nearswap.testnet
```

Comment:
```suggestion
export TOKEN1=token1.nearswap.testnet
```

Reply:
rest of the file should be updated accordingly.

Reply:
ok